src/AppBundle/Controller/RfXVoteCalculatorController.php

```diff
@@ -0,0 +1,221 @@
+            return $this->redirectToRoute(
+                "rfapResult",
+                $routeParams
+            );
+        } elseif ($projectQuery != "") {
+            return $this->redirectToRoute(
+                "rfapResult",
+                [
+                    'project'=>$projectQuery
+                ]
+            );
+        }
+
+        return $this->render(
+            'rfxVoteCalculator/index.html.twig',
+            [
+                "xtPage" => "rfap",
+            ]
+        );
+    }
+
+    /**
+     * Result View of RfXVoteCalculator
+     *
+     * @param string $project  The project we're working on
+     * @param string $username Username of the user we're analysing.
+     *
+     * @Route("/rfap/{project}/{username}", name="rfapResult")
+     *
+     * @return Response
+     */
+    public function resultAction($project, $username)
+    {
+        $api = $this->get("app.api_helper");
+
+        $conn = $this->getDoctrine()->getManager("replicas")->getConnection();
+
+        $projectData = ProjectRepository::getProject($project, $this->container);
+        $userData = new User($username);
+
+        $rfaParam = $this->getParameter("rfa");
+
+        if (!$projectData->exists() || $rfaParam == null) {
+            $this->addFlash("notice", ["invalid-project", $project]);
+            return $this->redirectToRoute("rfap");
+        }
+
+        $namespaces = $projectData->getNamespaces();
+
+        if (!isset($rfaParam[$projectData->getDomain()])) {
+            $this->addFlash("notice", ["invalid-project-cant-use", $project]);
+            return $this->redirectToRoute("rfap");
+        }
+
+        $pageTypes = $rfaParam[$projectData->getDomain()]["pages"];
+        $namespace
+            = $rfaParam[$projectData->getDomain()]["rfa_namespace"] !== null
+            ? $rfaParam[$projectData->getDomain()]["rfa_namespace"] : 4;
+
+        $finalData = [];
+
+        // We should probably figure out a better way to do this...
+        $ignoredPages = "";
+
+        if (isset($rfaParam[$projectData->getDomain()]["excluded_title"])) {
+            $titlesExcluded
+                = $rfaParam[$projectData->getDomain()]["excluded_title"];
+            foreach ($titlesExcluded as $ignoredPage) {
+                $ignoredPages .= "AND p.page_title != \"$ignoredPage\"\r\n";
+            }
+        }
+
+        if (isset($rfaParam[$projectData->getDomain()]["excluded_regex"])) {
+            $titlesExcluded
+                = $rfaParam[$projectData->getDomain()]["excluded_regex"];
+            foreach ($titlesExcluded as $ignoredPage) {
+                $ignoredPages .= "AND p.page_title NOT LIKE \"%$ignoredPage%\"\r\n";
+            }
+        }
+
+        foreach ($pageTypes as $type) {
+            $type = explode(":", $type, 2)[1];
+
+            $type = str_replace(" ", "_", $type);
+
+            $query = "SELECT DISTINCT p.page_namespace, p.page_title
+FROM `page` p
```

Comment:
I'm getting errors with this because the table names aren't fully qualified and the DB connection is defaulting to meta_p. These table names could be got with something like `$pageTable = $projectData->getRepository()->getTableName($projectData->getDatabaseName(), 'page');`.